src/ProgramBlockEditor.js

```diff
@@ -274,8 +274,11 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, Progra
             hasActionPanelControl && 'ProgramBlockEditor__program-block--pressed'
         );
         const ariaLabel = this.props.intl.formatMessage(
-            { id: `ProgramBlockEditor.command.${command}` },
-            { index: programStepNumber + 1 }
+            { id: `ProgramBlockEditor.command.${command.split(/([0-9]+)/)[0]}` },
```

Comment:
We don't need to do this `split()` as all the "ProgramBlockEditor.command.*" messages are the same. See the note under `messages.json`.